transport-native-epoll/src/main/c/io_netty_channel_epoll_Native.c

```diff
@@ -136,6 +151,21 @@ void throwOutOfMemoryError(JNIEnv* env) {
     (*env)->ThrowNew(env, exceptionClass, "");
 }
 
+int getSysctlValue(const char * property, int* returnValue) {
+    int rc=-1;
+    FILE *fd=fopen(property,"r");
+    if(fd != NULL) {
+      char buf[32]={0x0};
+      if(fgets(buf,32,fd) != NULL) {
+        *returnValue=atoi(buf);
+        rc = 0;
+      }
+      fclose(fd);
```

Comment:
fopen and fclose is std c functions and are portable. And any file open with fopen should get closed with fclose as its a file pointer and not file descriptor which close expects.
see http://stackoverflow.com/questions/19564797/difference-between-fclose-and-close